library/core/src/lib.rs

```diff
@@ -160,6 +160,10 @@
 #![feature(unboxed_closures)]
 #![feature(unsized_fn_params)]
 //
+// Extra-experimental language feature only used by core::simd
```

Comment:
I think we would want to either drop this from core::simd or inquire with the const development efforts as to the nature of the incompleteness. (By the time this PR lands, it may be a moot point, though, as the feature may well become complete...).

In particular, there's no current way to "silo" the incomplete_features allow to just core::simd, I believe, so this would effectively make all of core able to use this feature. We try to avoid doing that in libcore, AFAIK; incomplete features generally point at unsoundness or similar "bad bugs".

Reply:
@workingjubilee I missed this previously--why is this feature enabled in `core`?

This feature isn't required for the main `core::simd` functionality, it adds a handful of functions that we have feature-gated (and I assumed would be disabled in `core::simd`).

Reply:
They should be disabled, yes, but for some reason the crate requires the feature on when included as a module. I did not understand why.

Reply:
I realized I was mixing this up with a different feature.  I just entirely removed this feature in https://github.com/rust-lang/portable-simd/pull/161 (since it was only needed for the old shuffles, which used arrays as const generic params).  Perhaps we should merge that PR and update this PR with those changes.

Reply:
Oh, great! That simplifies things a lot.